src/index.js

```diff
@@ -573,6 +573,26 @@ class CoreConsoleAPI {
     })
   }
 
+  /**
+   * Get all Services available to an Organization
+   *
+   * @param {string} organizationId Organization ID
+   * @returns {Promise<Response>} the response
+   */
+  getServicesForOrg (organizationId) {
+    const sdkDetails = {}
```

Comment:
sdkDetails should contain the organizationId for debugging purposes

Reply:
ok good point thanks